components/omega/doc/devGuide/Docs.md

```diff
@@ -0,0 +1,57 @@
+(omega-dev-docs)=
+
+# Documentation
+
+The OMEGA documentation is generated using the
+[Sphinx](https://www.sphinx-doc.org/en/master/) package and is written in
+[MyST](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html)
+format.  We recommend these 
+[basic tips](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#roles-directives)
+on using MyST in Sphinx.
+
+Another easy way to get started is by taking a look at the existing source
+code for the documentation: 
+
+<https://github.com/E3SM-Project/Omega/tree/develop/components/omega/doc>
+
+Each time you add new features to OMEAG, the corresponding documentation must 
+be included with the pull request to add the code.  This includes documentation
+for both the User's Guide and the Developer's Guide. We will add some examples 
+as the documentation gets fleshed out.
+
+Please add anchors for internal linking to the top of every page before
+headings within the page whenever you think internal linking might be useful
+(now or in the future).  In the User's Guide, please start each anchor with
+`omega-`:
+
+```markdown
+(omega-quick-start)=
+
+# Quick Start for Users
+...
+```
+
+This is in anticipation of a future time when the OMEGA documentation might
+be combined with other components of E3SM.
+
+In the Developer's Guide, anchors should start with `omega-dev-`:
+
+```markdown
+(omega-dev-quick-start)=
+
+# Quick Start for Developers
+...
+```
+
+Documentation for an OMEGA feature in the User's Guide should contain 
+information that is needed for users who set up and run OMEGA, including:
+
+- Flags related to the feature that a user can (or must) set when building
+  OMEGA,
+- Config options that a user can modify related to the feature
+
+Documentation for the feature in the Developer's Guide may contain:
+
+- Relevant technical details about development specific to that feature
+- Details on unit tests that cover that feature
+- Details on polaris regression tests that cover that feature
```

Comment:
@vanroekel and @philipwjones, this is by no means a complete or exhaustive list.  Feel free to suggests changes and additions either now or later when we have more experience and more examples to work from.

Reply:
User guide scope is about right - might need at least a brief explanation of a feature to know what the config options mean.

The developer guide should also serve as a reference manual, so should describe not only the actual mathematical terms (or a reference to them) but also the discretized form with variables close to what are used in the code. This provides an easy way for other developers or reviewers to verify the code against the intended implementation

Reply:
> might need at least a brief explanation of a feature to know what the config options mean.

I'm not sure what term you want to use.  These are what would be namelist options for a Fortran code but are now in the YAML file.

Reply:
Your suggestions for the developer's guide are great!

Reply:
Ah, I see. I'll add that. I completely agree.

Reply:
Done.